repose-aggregator/components/filters/keystone-v2-filter/src/main/scala/org/openrepose/filters/keystonev2/KeystoneRequestHandler.scala

```diff
@@ -21,11 +21,11 @@ package org.openrepose.filters.keystonev2
 
 import java.io.InputStream
 import java.util.{Calendar, GregorianCalendar}
-import javax.servlet.http.HttpServletResponse._
-import javax.ws.rs.core.{HttpHeaders, MediaType}
 
 import com.fasterxml.jackson.core.JsonProcessingException
 import com.typesafe.scalalogging.slf4j.StrictLogging
+import javax.servlet.http.HttpServletResponse._
+import javax.ws.rs.core.{HttpHeaders, MediaType}
```

Comment:
Some of us must have our import placement rules different from each other, because this is a second class where these moved.

Reply:
I think my are default, but we should probably check.